test/infra/cdn-site-hosting/cdn-site-hosting-construct.test.ts

```diff
@@ -82,4 +82,49 @@ describe("CdnSiteHostingConstruct", () => {
       );
     });
   });
+
+  describe("For a routed SPA", () => {
+    let stack: Stack;
+
+    beforeAll(() => {
+      const app = new cdk.App();
+      stack = new cdk.Stack(app, "TestRoutedSPAStack", { env: testEnv });
+      new CdnSiteHostingConstruct(stack, "MyTestConstruct", {
+        certificateArn: fakeCertificateArn,
+        siteSubDomain: fakeSiteSubDomain,
+        domainName: fakeDomain,
+        removalPolicy: RemovalPolicy.DESTROY,
+        isRoutedSpa: true,
+        sources: [s3deploy.Source.asset("./")],
+        websiteErrorDocument: "",
```

Comment:
This is not generally how you'd specify omitting this, is it? Does it suggest you need to make the property allow `undefined`?

Reply:
Can you include a test where you provide a non-falsy value here, please?

Reply:
I did think about changing it but didn't because I assumed the the type would already allow undefined if CDK allowed it, but happy to update :)

Reply:
The type for the props is defined in this project, regardless what the underlying constructs support - you'll need to change the props definition to drop it, but that seems reasonable to me.